conf/airframes/AGGIEAIR/aggieair_rp3_lia.xml

```diff
@@ -35,6 +35,8 @@ RP3 Lisa MX
 
     <define name="AGR_CLIMB" />
     <define name="POLY_OSAM_HALF_SWEEP_ENABLED" value="FALSE"/>
+    <define name="AIRSPACE"/>
+    <define name="DATALINK_LOST_TIME" value="45"/>
```

Comment:
There is an exception that can be defined in the flight plan, this is if you want to enforce datalink check independent on the flight plan. Should be more clear with the updated commit.